models/CoronSegmentator/scripts/cardiac_segmentation/auto3dseg_segresnet_inference.py

```
@@ -0,0 +1,291 @@
+                allow_missing_keys=True,
+            ),
+        ]
+        _add_normalization_transforms(
+            ts, "image", main_normalize_mode, intensity_bounds
+        )
+    else:  # multiple input images
+        ts = []
+
+        extra_modalities = OrderedDict(config["extra_modalities"])
+        normalize_modes = [main_normalize_mode] + list(extra_modalities.values())
+        for key, normalize_mode in zip(keys, normalize_modes):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)